docs/how-to-deploy-a-bundle.md

```diff
@@ -0,0 +1,74 @@
+# Creating and Installing a Custom Build
+
+You can build a cluster-logging-operator directly from source code,
+and create a bundle image that will allow you to deploy it to your cluster.
+
+You need to create two images:
+
+* operator image: Contains your build of the operator.
+* bundle image: Contains instructions for OLM to install you operator.
+
+## Log-in to Image Repositories
+
+You need a public image registry, for example `quay.io/somebody`, to _write_ your operator and bundle images.
+
+**NOTE**: The images you create must be _public_ so your cluster will be able to pull them.
+For example, on `quay.io` you need to mark images as public in the "organization settings".
+
+This document will use `$REGISTRY` to refer to your image registry.
+
+You also need to log in to the red Hat registry for related images.
+Use your Red Hat customer portal username and password.
+See https://access.redhat.com/RegistryAuthentication for more.
+
+``` bash
+podman login $REGISTRY
+podman login registry.redhat.io
+```
+
+Note: this document uses `podman` but you can substitute `docker` instead.
+
+## Build and Push Operator and Bundle Images
+
+Clone the cluster-logging-operator repository:
+
+``` bash
+git clone http://github.com/openshift/cluster-logging-operator
+cd cluster-logging-operator
+```
+
+Build and push the operator image:
+
+``` bash
+make image IMAGE_TAG=$REGISTRY/my-cluster-logging-operator:latest
+podman push $REGISTRY/my-cluster-logging-operator:latest
+```
+
+Build and push the bundle image:
+
+``` bash
+make bundle
```

Comment:
How do we build a specific bundle that points to specific operand images?

Reply:
Create a custom overlay config/overlays/myoverlay and run make with OVERLAY=config/overlays/myoverlay
This uses kustomize to modify the resources in config before making the bundle.